sklearn/model_selection/_search_successive_halving.py

```diff
@@ -203,11 +189,6 @@ def _check_input_parameters(self, X, y, groups):
                 "an empty dataset X."
             )
```

Comment:
I can't suggest, but I think the condition just above here must be removed now that it's tested via parameters' validation.

Reply:
`min_resources_` is a validated parameter: it can depend on the value of `max_samples` or a string that depends on `X`. So it could happen to be 0 for some corner cases.

It is different from validating `min_resources` which is the direct input parameter.